script.module.pyrollbar/addon.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pyrollbar"
+       name="pyrollbar"
+       version="0.15.2"
+       provider-name="rollbar">
```

Comment:
This looks like a modified version of the module. Can you add yourself as an author?

Reply:
Yes it is a straight copy of the pyrollbar package plus an additional module to simplify its use within Kodi. I've added my name as an additional author.